taxi_service/urls.py

```diff
@@ -1,21 +1,9 @@
-"""taxi_service URL Configuration
-
-The `urlpatterns` list routes URLs to views. For more information please see:
-    https://docs.djangoproject.com/en/4.0/topics/http/urls/
-Examples:
-Function views
-    1. Add an import:  from my_app import views
-    2. Add a URL to urlpatterns:  path('', views.home, name='home')
-Class-based views
-    1. Add an import:  from other_app.views import Home
-    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
-Including another URLconf
-    1. Import the include() function: from django.urls import include, path
-    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
-"""
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
+
+import taxi
 
 urlpatterns = [
     path("admin/", admin.site.urls),
+    path("", include("taxi.urls", namespace="taxi"))
```

Comment:
what about this part ` + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)` ?